src/views/Landing.vue

```diff
@@ -131,29 +131,29 @@
           >
             <VueAos animation-class="animate__animated animate__delay-1s animate__fadeInUp">
               <div class="sh-col-3 sm-text-box">
-                <h4>{{ stats.total_tips_length }}</h4>
+                <h4>{{ stats.totaltipslength }}</h4>
```

Comment:
naming gets more unusual 🙃

Reply:
Yeah... Not what I wish, but the return value from materialized views are forced lowercase

Reply:
We could map it to different names in backend, we should probably discuss that with the team

Reply:
Its fixed now :)